### ISSUE TITLE
rust version

### ISSUE THREAD

**Issue by ondohotola** [jqnatividad/qsv#1040]
Hi,

I am getting this on my M2

```
package `qsv-sniffer v0.9.4` cannot be built because it requires rustc 1.70.0 or newer, while the currently active rustc version is 1.69.0
```

can you perhaps allow rust 1.69 for the time being, ie until Homebrew updates itself?

**Comment by jqnatividad** [comment]
I'm afraid it's not that simple as I pretty much upgraded across the board all the qsv suite crates to use OnceLock which was stabilized in 1.70.0.

I tried waiting for homebrew to upgrade to 1.70.0, and it usually only lags a few days, but it's taking longer than usual.  I'm still optimistic it'll be upgraded this week.

**Comment by ondohotola** [comment]
Thanks,

it's a bit of a nuisance as I only have acces to my M2 laptop (in
transit) and wanted to make use of the new features.

So, patience it is.

**Comment by jqnatividad** [comment]
You can also always install rust directly, instead of using homebrew.

**Comment by ondohotola** [comment]
Sure, but that complicates things.

Patience is a virtue :-)-O

**Comment by jqnatividad** [comment]
😄  Tracking https://github.com/Homebrew/homebrew-core/pull/132584 and will not another release until its homebrew upgrades rust to 1.70.0

**Comment by chenrui333** [comment]
We shipped the 1.70.0 release on the homebrew side, this issue can be closed now.
